installation/scripts/testing.sh

```diff
@@ -26,12 +26,15 @@ metadata:
   name: ${suiteName}
 spec:
   maxRetries: 1
-  concurrency: 1
+  concurrency: 2
```

Comment:
no, no, no - we have to know which tests can be run concurrently and only after that we can enable concurrency